src/towncrier/_project.py

```diff
@@ -50,8 +50,12 @@ def get_version(package_dir: str, package: str) -> str:
     if isinstance(version, str):
         return version.strip()
 
-    if isinstance(version, Version):
-        return version.base().strip()
+    if isinstance(version, IncrementalVersion):
+        # FIXME:https://github.com/twisted/incremental/issues/81
```

Comment:
Not sure if this hack is ok... I am not happy with it.. but at the same time I don't know what to do.

I would rather have a green test, than remove the test.